packages/block-editor/src/components/plain-text/index.js

```diff
@@ -10,15 +10,41 @@ import { forwardRef } from '@wordpress/element';
 import TextareaAutosize from 'react-autosize-textarea';
 import classnames from 'classnames';
 
+/**
+ * Constants
+ */
+const REGEXP_NEWLINES = /[\r\n]+/g;
+
 /**
  * @see https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/plain-text/README.md
  */
-const PlainText = forwardRef( ( { onChange, className, ...props }, ref ) => {
+const PlainText = forwardRef( ( {
+	onChange,
+	className,
+	multiline,
```

Comment:
I think this should probably be on by default, I also think we may want to check if RichText has a similar prop (to align names).

Reply:
`RichText` does have [`multiline` ](https://github.com/WordPress/gutenberg/blob/5d0c03e39ecd69ee814bb7f938373bfee96a6928/packages/block-editor/src/components/rich-text/index.js#L74) (although the meaning is not really the same), reason why I've used it here.

I personally find props that default to true to be a bit awkward to use, but I don't have a strong opinion about it.

Reply:
> I personally find props that default to true to be a bit awkward to use, but I don't have a strong opinion about it.

But if it was the previous default behavior, isn't it more about preserving backward compatibility?

Reply:
Makes sense @aduth.
I've updated `multiline` to default to `true` and dropped the changes from the other `PlainText` users. 👍